src/Microsoft.AspNetCore.Diagnostics.HealthChecks/HealthCheckMiddleware.cs

```diff
@@ -70,6 +71,13 @@ public async Task InvokeAsync(HttpContext httpContext)
 
             httpContext.Response.StatusCode = statusCode;
 
+            if (!_healthCheckOptions.SuppressCacheHeaders)
```

Comment:
I considered also a design where you could give us the value you wanted to return from `Cache-Control`, but needing to support `Pragma` complicates things.

Some h/c systems are HTTP 1.0 so this is an area where we **do** care.